README.md

```diff
@@ -1 +1,228 @@
-# hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="./luis-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My new Blog</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My new Blog <a name="about-project"></a>
+
+> Hello Microvers, I created it using HTML, CSS, and JavaScript
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+1-HTML.
+2-CSS.
+3-Liners.
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li> 
+  </ul>
+</details>
+
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+-Added index.html file.
+-Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Comming soon!!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+It Will be great for you and your friends!!!
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1-Web browser.
+2-Code Editor.
+3.Git-smc.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/lerfast/my-new-blog.git
+```
+
+### Install
+
+Install this project with:
+
+Run command:
+
+```sh
+  cd my-project
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if your cloning command with the correct link and the accessing cloned directory command to make your documentation be more descriptive and professional. i.e
```sh

## Bad
  ...
  git clone https://github.com/lerfast/my-new-blog.git

  ....
  cd my-project
------------------------------------------------------------------
## Good

...
  git clone https://github.com/lerfast/hello-world.git

  ....
  cd hello-world
```